janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphPerformanceMemoryTest.java

```diff
@@ -45,6 +50,35 @@
     @Rule
     public TestRule benchmark = JUnitBenchmarkProvider.get();
 
+    @Test
+    void labelRetrivial() {
+        // Fill added relations container
+        for (int i = 0; i < 10000; i++) {
+            graph.traversal()
+                 .addV("V1")
+                 .property("p1", i)
+                 .property("p2", i)
+                 .property("p3", i)
+                 .property("p4", i)
+                 .property("p5", i)
+                 .property("p6", i)
+                 .property("p7", i)
+                 .property("p8", i)
+                 .property("p9", i)
+                 .iterate();
+        }
+        // Try access to label in same tx
+        List<Vertex> vertices = graph.traversal()
+                                     .V()
+                                     .toList();
+        long start = System.currentTimeMillis();
+        for (int i = 0; i < 100; i++) {
+            vertices.forEach(v -> v.label());
+        }
+        graph.tx().rollback();
+        System.out.println(System.currentTimeMillis() - start);
```

Comment:
Why a `println` in a test?

Reply:
Another tests in this class used `println` for writing some info

Moreover this test has `MEMORY_TESTS` category that activated by memory-test profile, but this profile not activated in travis, so it may be outdated test. I think JG need to rework performance/memory tests for proper checking regression in this space